### ISSUE TITLE
brw(import) seems to always produce an error in expansion

### ISSUE THREAD

**Issue by Gorialis** [jam1garner/binrw#123]
The following:
```rust
use binrw::*;

#[derive(Clone)]
#[binrw]
#[brw(import { threshold: u8 })]
struct Blah {}
```
produces this error:
![image](https://user-images.githubusercontent.com/8049998/168717045-9eba7b55-a379-47b3-ab2c-bdedb7653b1d.png)

This error appears to be caused by the import line, and goes away if you adjust it to either `#[bw(import { threshold: u8 })]` or `#[br(import { threshold: u8 })]`. This doesn't work for my use case, since both the reading and writing require state that isn't determinable statically.

Note that the same thing occurs if you include both `bw` and `br` instead of one `brw`, and it doesn't go away if you use a different struct or arg names between `br` and `bw`.
(*edit*: It also doesn't go away if you don't derive Clone)

This is with binrw `0.8.4` (checksum `a13d2fa8772b88ee64a0d84602c636ad2bae9c176873f6c440e2b0a1df4c8c43`), rustc 1.60.0 with either the stable or nightly toolchain.

**Comment by jam1garner** [comment]
yep, looks like a clear cut copy/paste error. My bad! thank you for the report. I'll try to get a fix out soon

**Comment by jam1garner** [comment]
actually wait I think this is already fixed on master, if you need a prerelease of 0.9.0 pushed to crates.io feel free to @ me.

**Comment by Gorialis** [comment]
This does indeed seem to be fixed on master. Thanks!
